# 12 Cushing days of cover

Does the gap between five-print RWTC returns when Cushing days of cover are below the pre-2020 20th percentile, and when they are not, keep its sign after 2019?

Holdout starts 2020-01-01. Verdict: held.

Cushing crude stocks (thousand barrels) divided by US refiner net inputs of crude (thousand barrels per day) on the same Friday. The result is days. Low cover is a print below the train 20th percentile, frozen at 1.403 days.

In [1]:
import json
from pathlib import Path
studies = json.loads(Path('data/studies.json').read_text())
study = next(item for item in studies['studies'] if item['slug'] == 'cushing-cover')
print(study['verdict'])
print(study['verdict_line'])
print(study['table']['caption'])
for row in study['table']['rows']:
    print(' | '.join(row))


held
Train 20th percentile of days of cover is 1.403. Train gap, low cover minus the rest, is -0.204% (145 low, 581 rest). Holdout gap is -0.319% (60 low, 290 rest). The holdout kept the train sign.
Five-print RWTC log return when days of cover are below the frozen train 20th percentile
Train | 145 | 581 | -0.204%
Holdout | 60 | 290 | -0.319%


## Method

The price date is the Friday print when RWTC exists that day, otherwise the next RWTC print within 3 calendar days. The return then steps five published RWTC prints.

## Result

Train 20th percentile of days of cover is 1.403. Train gap, low cover minus the rest, is -0.204% (145 low, 581 rest). Holdout gap is -0.319% (60 low, 290 rest). The holdout kept the train sign.